OKiMaureen/favorite-things
# #167829383 Create favourites components and functionalities

[Finishes]#167829383

#### What does this PR do?

- create view favourite  things component and functionalities


#### Description of Task to be completed?
 - create view favourites component and functionality
- craete edit favourite component and functionality
- create update favourite component and functionality
- creata favoutite component and functionality
- create delete favourite component and functionality

#### How should this be manually tested?
- Fetch and check out to this branch with `git fetch   ft-create-view-favourites-167829383  && git checkout  f ft-create-view-favourites-167829383`
- Open Browser and make a  request to `http://127.0.0.1:8080/favourites`. 

#### What are the relevant pivotal tracker stories?
[#167829383(https://www.pivotaltracker.com/story/show/#167829383) 

[Finishes #167829383]
